python/lsst/cp/pipe/cpFunctions.py

```diff
@@ -124,6 +115,12 @@ def registerCalibrations(self, datasetTypeName):
         self.butler.collection = None
 
         for newId, data in zip(self.newDataId, self.objects):
+            # Special case known special storageClasses.
```

Comment:
Can I suggest putting this in a method so it's easy to supplement or override? This will make it easier to add new calibs in the future.